spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/DocServiceConfigurator.java

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2019 LINE Corporation
+
+LINE Corporation licenses this file to you under the Apache License,
+version 2.0 (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at:
+
+  https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+License for the specific language governing permissions and limitations
+under the License.
+*/
+
+package com.linecorp.armeria.spring;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.AnnotationAwareOrderComparator;
+import org.springframework.core.annotation.Order;
+
+import com.linecorp.armeria.server.docs.DocServiceBuilder;
+
+@FunctionalInterface
+public interface DocServiceConfigurator extends Ordered {
```

Comment:
Add documentation for interface?

Reply:
I added the documentation for interface, if it needs to be changed let me know